### PR TITLE
Possible override of runLocally task with timeout parameter optional.

### PR DESCRIPTION
**Pull request by EkAndreas**
Regarding issue with timeout running local tasks that requires long execution.
This is a fix that allows the command to be attached with a timeout parameter optionally.

### CODE REVIEW

File: src/functions.php [hunk 1]
```diff
@@ -207,12 +207,14 @@ function run($command)
 /**
  * Execute commands on local machine.
  * @param string $command Command to run locally.
+ * @param int $timeout (optional) Override process command timeout in seconds.
  * @return string Output of command.
  * @throws \RuntimeException
  */
-function runLocally($command)
+function runLocally($command, $timeout=60)
```
Comment (elfet) [inline src/functions.php]: `$timeout=60` to `$timeout = 60`.

File: src/functions.php [hunk 2]
```diff
@@ -207,12 +207,14 @@ function run($command)
...
 {
     $process = new Symfony\Component\Process\Process($command);
+	$process->setTimeout( $timeout );
```
Comment (elfet) [inline src/functions.php]: 4 spaces instead of tab. 
`setTimeout( $timeout );` to `setTimeout($timeout);`

**Comment by EkAndreas** [comment]
fixing spaces

File: src/functions.php [hunk 3]
```diff
@@ -207,12 +207,14 @@ function run($command)
 /**
  * Execute commands on local machine.
  * @param string $command Command to run locally.
+ * @param int $timeout (optional) Override process command timeout in seconds.
  * @return string Output of command.
  * @throws \RuntimeException
  */
-function runLocally($command)
+function runLocally($command, $timeout = 60 )
```
Comment (GrahamCampbell) [inline src/functions.php]: no space after 60 surely?

**Comment by EkAndreas** [comment]
Sorry, I'm new to pull requests... thanks!

File: src/functions.php (hunk 3 above)
Comment (EkAndreas) [inline src/functions.php]: ok, thanks!

File: src/functions.php [hunk 4]
```diff
@@ -207,12 +207,14 @@ function run($command)
...
  * @param string $command Command to run locally.
+ * @param int $timeout (optional) Override process command timeout in seconds.
  * @return string Output of command.
  * @throws \RuntimeException
  */
-function runLocally($command)
+function runLocally($command, $timeout = 60)
 {
     $process = new Symfony\Component\Process\Process($command);
+    $process->setTimeout( $timeout );
```
Comment (GrahamCampbell) [inline src/functions.php]: No spaces around this too. I'm only assuming we're using PSR-2 here. :)

Comment (EkAndreas) [inline src/functions.php]: Lol! Thanks! My PHPStorm isn't configured for PSR-2 yet. Using some kind of WordPress formatter :-)

**Comment by elfet** [comment]
Thanks!

**Pull request merged**